app/views/participants/validations/manual_details_check_fip.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title = "We’re checking your details" %>
+<% content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_panel title: title, body: nil, classes: "govuk-!-margin-bottom-8" %>
+    <h2 class="govuk-heading-m">What happens next</h2>
```

Comment:
Ditto, h1

Reply:
As above, `govuk_panel` renders the `title` as a H1 element

Reply:
Ah, alright, forgot about that. Nice :)